README.md

```diff
@@ -113,6 +121,10 @@ fn main() -> PyResult<()> {
 }
 ```
 
+If you want to excute a code with multiple lines, you can use
```

Comment:
I don't think "multiple lines" is quite accurate here: the difference is between executing a *single expression* (and getting the return value), and executing *one or more statements*.

For example, although it's a single line, you can `eval("assert True")`.

Reply:
Thanks.